lib/contentful/bootstrap/commands.rb

```diff
@@ -105,6 +106,29 @@ def generate_token(space, options = {})
         token
       end
 
+      def generate_json(space_id, options = {})
+        filename = options.fetch(:filename, space_id)
+        access_token = options.fetch(:access_token)
+
+        if access_token.nil?
+          puts "Access Token not specified"
+          puts "Exiting!"
+          exit
+        end
+
+        puts "Generating JSON Template '#{filename}' for Space: '#{space_id}'"
+        puts "If you've indicated a filename, JSON will be automatically saved to that file"
+        puts
```

Comment:
I think it would be preferable to have different messages when writing to a file or to stdout.